Library/Formula/slimerjs.rb

```diff
@@ -11,9 +31,12 @@ class Slimerjs < Formula
     sha1 "aa5654afdd8d2dc049878d2898b9fb7fa33911ab" => :mountain_lion
   end
 
-  if MacOS.version > :snow_leopard
-    option "without-xulrunner", "Build without xulrunner (requires a installed Firefox)"
-    depends_on "xulrunner" => :recommended
+  option "with-xulrunner", "Build with xulrunner"
+
+  if build.with? "xulrunner" && MacOS.version > :snow_leopard
```

Comment:
Presumably, a mandatory Firefox dependency, from the current wording. Does the current wording in this PR catch that or is it shielded by the ` MacOS.version ` conditional?